file/Tiltfile

```diff
@@ -0,0 +1,21 @@
+def read_file(path):
```

Comment:
hmmm...`read_file` is already a builtin, so i'm not sure what this adds - https://docs.tilt.dev/api.html#api.read_file

Reply:
Its a bad MR i guess